### PR TITLE
Serialize and seed lesson-standard relationships

### PR DESCRIPTION
**Pull request by davidsbailey**
Continues [PLAT-751]. Seeding of standards themselves is handled by previous PRs. This adds serialization and seeding only for lesson-standard relationships. For more background, see [Lesson Standards design doc](https://docs.google.com/document/d/1zvdv7y8VuRz8KqT1rzqeJAmvirCvUr1jS8OWy0p9FHk/edit)

## Testing story

updates existing unit test coverage, and adds new unit test coverage for adding and removing lesson-standard relationships. 

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

[PLAT-751]: https://codedotorg.atlassian.net/browse/PLAT-751

### CODE REVIEW

File: dashboard/test/lib/services/script_seed_test.rb [hunk 1]
```diff
@@ -75,14 +75,16 @@ class ScriptSeedTest < ActiveSupport::TestCase
       #   16 queries, one for each LevelsScriptLevel.
       #   4 queries, one to remove LessonsResources from each Lesson.
       #   2 queries, one to remove LessonsVocabularies from each Lesson.
+      #   2 queries, one to remove LessonsProgrammingExpression from each Lesson.
```
Comment (dmcavoy) [inline dashboard/test/lib/services/script_seed_test.rb:79]: Thanks for fixing!

File: dashboard/test/lib/services/script_seed_test.rb [hunk 2]
```diff
@@ -601,7 +635,7 @@ class ScriptSeedTest < ActiveSupport::TestCase
     # Programming Expressions are shared across all courses. We need to make sure all the
     # programming expressions we need for this script are created, but we should never remove
     # a programming expression because it might be in use by another script.
-    test 'seed deletes lessons_standards' do
+    test 'seed deletes lesson programming expressions' do
```
Comment (dmcavoy) [inline dashboard/test/lib/services/script_seed_test.rb:635]: Thanks again

**Review by dmcavoy — APPROVED** [review]

**Comment by davidsbailey** [comment]
PTAL @dmcavoy  -- I had to add some interesting bits to include the framework in the seeding key. I think you'll need to do something similar for programming expressions, to make sure that ambiguous expression names are resolved correctly.

**Comment by dmcavoy** [comment]
@davidsbailey I see what you had change and that makes complete sense to me. I will follow up and do the same for the programming expressions. This PR looks good to go from my perspective

File: dashboard/test/lib/services/script_seed_test.rb [hunk 3]
```diff
@@ -342,6 +345,8 @@ class ScriptSeedTest < ActiveSupport::TestCase
 
     test 'seed updates lesson programming expressions' do
       script = create_script_tree
+      CourseOffering.add_course_offering(script)
```
Comment (dmcavoy) [inline dashboard/test/lib/services/script_seed_test.rb]: Why does this test need course offering?

Comment (davidsbailey) [reply dashboard/test/lib/services/script_seed_test.rb]: This was an unintentional result of a bad merge. I'm not really sure whether it is best to include it or remove it. It isn't necessary for the test to pass, so I've removed it. Probably this means the script being tested will have no resources or vocab, so we aren't testing that resources and vocab are preserved, but that seems a bit weirdly specific to worry about testing here.

**Pull request merged**